### PR TITLE
Move client/web/src/auth/Steps to wildcard/components folder #25244

### PR DESCRIPTION
**Pull request by saurabh042**
Move client/web/src/auth/Steps to wildcard/components folder #25244

### CODE REVIEW

File: client/web/src/auth/PostSignUpPage.tsx [hunk 1]
```diff
@@ -20,7 +20,7 @@ import { UserExternalServicesOrRepositoriesUpdateProps } from '../util'
 import styles from './PostSignUpPage.module.scss'
 import { getReturnTo } from './SignInSignUpCommon'
 import signInSignUpCommonStyles from './SignInSignUpCommon.module.scss'
-import { Steps, Step, StepList, StepPanels, StepPanel, StepActions } from './Steps'
+import { Steps, Step, StepList, StepPanels, StepPanel, StepActions } from '@sourcegraph/wildcard/'
```
[the remark this answers is not in the document]
Comment (valerybugakov) [reply client/web/src/auth/PostSignUpPage.tsx]: @saurabh042, nice! 
The `eslint` check failed on CI. Could you run `yarn all:eslint` locally, fix reported errors, and push changes?

[the remark this answers is not in the document]
Comment (saurabh042) [reply client/web/src/auth/PostSignUpPage.tsx]: Hi  @valerybugakov when i am trying to run yarn all:eslint i am getting below error:
`yarn run v1.22.15
$ DOCSITE_LIST="$(./dev/docsite.sh -config doc/docsite.json ls)" dev/foreach-ts-project.sh yarn -s run eslint --quiet
'DOCSITE_LIST' is not recognized as an internal or external command,
operable program or batch file.
error Command failed with exit code 1.`

Could you please help in resolving this?

[the remark this answers is not in the document]
Comment (valerybugakov) [reply client/web/src/auth/PostSignUpPage.tsx]: @saurabh042, would the following command work for you?

```sh
dev/foreach-ts-project.sh yarn run eslint

[the remark this answers is not in the document]
Comment (saurabh042) [reply client/web/src/auth/PostSignUpPage.tsx]: @valerybugakov It doesn't work either.

$ ./dev/foreach-ts-project.sh yarn run eslint
--- client/web: yarn run eslint
+ cd client/web
+ eval ' yarn run eslint'
++ yarn run eslint
yarn run v1.22.15
$ eslint --cache '**/*.[tj]s?(x)'
' was unexpected at this time.

D:\playground\hactoberfest\sourcegraph\client\web>  "D:\playground\hactoberfest\sourcegraph\node_modules\.bin\\node.exe"  "D:\playground\hactoberfest\sourcegraph\node_modules\.bin\\..\eslint\bin\eslint.js" --cache '**/*.[tj]s?(x)'
error Command failed with exit code 255.

[the remark this answers is not in the document]
Comment (valerybugakov) [reply client/web/src/auth/PostSignUpPage.tsx]: @saurabh042, would be running `yarn eslint` in each `./client` subfolder work for you?

[the remark this answers is not in the document]
Comment (saurabh042) [reply client/web/src/auth/PostSignUpPage.tsx]: @valerybugakov I am still getting error on running `yarn eslint` in the `./client` subfolders. Pasting the error below:
 "D:\playground\hactoberfest\sourcegraph\node_modules\.bin\\node.exe"  "D:\playground\hactoberfest\sourcegraph\node_modules\.bin\\..\eslint\bin\eslint.js" --cache '**/*.[jt]s?(x)'
error Command failed with exit code 255.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

Is it possible that this error message is OS dependent? I am running it on windows.

[the remark this answers is not in the document]
Comment (valerybugakov) [reply client/web/src/auth/PostSignUpPage.tsx]: It's possible. I found a similar issue here: https://github.com/eslint/eslint/issues/9353.
Researching with Google using keywords `yarn run eslint windows exit 255` should help.

**Review by pdubroy — COMMENTED** [review]
From my side, this looks good — Valery, what do you think?

**Comment by eseliger** [comment]
As part of #wg-delete-week I am going to close this PR. If it’s still needed, feel free to reopen and restore the branch.

**Pull request closed without merging**